lib/statsd.js

```diff
@@ -311,12 +311,9 @@ Client.prototype.send = function (message, tags, callback) {
     throw this.dnsError;
   }
 
-  var mergedTags = [];
+  var mergedTags = this.globalTags;
   if(tags && Array.isArray(tags)){
-    mergedTags = mergedTags.concat(tags);
-  }
-  if(this.globalTags && Array.isArray(this.globalTags)){
-    mergedTags = mergedTags.concat(this.globalTags);
+    mergedTags = overrideTags(mergedTags, tags);
```

Comment:
Yes, it'll change the behaviour but I did it intentionally. I think it makes sense.